src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -562,8 +583,18 @@
     </Touch>
   </Target>
 
+  <Target Name="DiscoverSymbolsArchiveFile"
+          Condition="'$(DotNetBuildSourceOnly)' == 'true' and
+                     '$(IsUtilityProject)' != 'true' and
+                     Exists('$(RepoAssetsSymbolsArchive)')">
+
+    <ItemGroup>
+      <RepoSymbolsArchive Include="$(RepoAssetsSymbolsArchive)" />
+    </ItemGroup>
+  </Target>
```

Comment:
I would remove this target as it only sets a single item.

Reply:
I'll see if this can be merged with the target that consumes the `RepoSymbolsArchive` or move it outside of target (and delete it).

Reply:
Oh I now understand why this is necessary. It's used as an input and inside the condition of `ExtractSymbolsArchiveFile`. That target then makes sense to me.